packages/forma-36-react-components/src/components/Tag/Tag.tsx

```diff
@@ -5,29 +5,28 @@ import styles from './Tag.css';
 
 export type TagType =
   | 'primary'
+  | 'filled-primary'
```

Comment:
maybe `primary-filled` to keep the element-variant structure?

Reply:
@mshaaban0 sure, that was my first thought, but somehow it sounded strange. But yeah, this makes more sense! on my way :)